contrib/sly-package-fu.el

```diff
@@ -144,9 +155,12 @@ places the cursor at the start of the DEFPACKAGE form."
                                  (1+ (point))
                                (point))))))))))))
 
-(defun sly-export-symbols ()
+(defun sly-read-symbols ()
```

Comment:
Please rename this to `sly-package-fu--read-symbols` and tweak the docstring to state that it reads a list of symbols following point and is intended to be used inside defpackage forms that list symbols.

Reply:
Done